lib/order.rb

```diff
@@ -0,0 +1,68 @@
+require_relative 'menu'
+require_relative 'confirmation'
+require 'dotenv'
+Dotenv.load
+
+class Order
+
+  attr_reader :selection, :menu, :prices, :total, :orders, :confirmation
+
+  def initialize(menu = Menu.new)
+  @menu = menu
+  @orders = []
+  @selection = {}
+  @prices = []
+  end
+
+  def start
+    puts @menu.introduction
+    puts @menu.show_menu
+  end
+
+  def select_order(item, amount)
+    @selection = { item.to_sym => amount}
+    add_to_orders
+    return "#{amount} x #{item} added to your order!"
+  end
+
+  def get_price
+    @orders.each do |order|
+    order.each do |item, amount|
+    @price = menu.dishes[item]
+    puts "#{amount} x #{item.to_sym} at £#{@price} each = £#{@price *= amount}"
+    @prices << @price
+  end
+  end
+  end
+
+  def calculate_price
+    total_price
+    puts display_total_price
+    puts confirm_alert
+  end
+
+  def total_price
+    @total = @prices.inject { |accum_value, element| accum_value += element }
```

Comment:
This line is really easy to understand because of it uses Ruby's built-in inject, rather than using custom code to do the summing.